pkg/asset/machines/worker.go

```diff
@@ -319,6 +319,15 @@ func (w *Worker) Generate(ctx context.Context, dependencies asset.Parents) error
 				}
 				machineConfigs = append(machineConfigs, ignPowerSMT)
 			}
+
+			if installConfig.Config.Publish == types.InternalPublishingStrategy &&
+				(len(installConfig.Config.ImageDigestSources) > 0 || len(installConfig.Config.DeprecatedImageContentSources) > 0) {
+				ignChrony, err := machineconfig.ForCustomNTP("worker", "time.adn.networklayer.com")
```

Comment:
Can we create a constant that is common to master and worker and use that here in place of hardcoding in master and worker machine config paths separately?

Reply:
Now defined as a constant in `defaults` package.